src/components/Match/Vision/index.jsx

```diff
@@ -82,7 +82,7 @@ const Styled = styled.div`
   }
 }
 `;
-
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
we should just make this a warning in eslint config

Reply:
normally it is a warning, but in CI build it fails beacause warnings are treated as errors. Interesting is, that it only fails at this warning, there are several more in the project which build ignores.